india_compliance/gst_india/overrides/payment_entry.py

```diff
@@ -14,6 +16,40 @@
 from india_compliance.gst_india.utils import get_all_gst_accounts
 
 
+@frappe.whitelist()
+def get_reconciliation_status_for_invoice_list(invoice_list):
+    if not invoice_list:
+        return
+
+    if isinstance(invoice_list, str):
+        invoice_list = json.loads(invoice_list)
+
+    original_list = frappe.db.get_list(
```

Comment:
Use `frappe.get_list`. Note: `get_list` checks permissions implicitly, if you intend to not check those, you must specify the reason and then use `frappe.get_all` instead.